cstrml/market/src/lib.rs

```diff
@@ -141,19 +155,29 @@ impl<T: Trait> MarketInterface<<T as system::Trait>::AccountId,
     fn maybe_set_sorder(order_id: &<T as system::Trait>::Hash,
                         so: &StorageOrder<<T as system::Trait>::AccountId>) {
         Self::maybe_set_sorder(order_id, so);
+        T::Payment::start_delayed_pay(order_id);
```

Comment:
Is this correct? cause `maybe_set_sorder` can be called when `order_status` from `pending` to `success` or from `success` to `failed`, or we will add more status like `closed`?